tests/e2e/README.md

```diff
@@ -96,14 +113,15 @@ make push
 
 ```
 
-**Option 2:** Already committed changes to istio/istio master branch
+**Option 3:** Already committed changes to istio/istio master branch
 NOTE: SHA used as TAG is one that is already committed on istio/istio. You can pick any SHA you want.
 ```
+export LOCALREG=false
 export HUB="gcr.io/istio-testing"
 export TAG="d0142e1afe41c18917018e2fa85ab37254f7e0ca"
 ```
 
-**Option 3:** Testing local changes
```

Comment:
Should your option replace this completely?

What's the point of uploading local changes to public registry?